neofetch

```diff
@@ -3549,16 +3549,26 @@ END
                 # like a font definition. NOTE: There is a slight limitation in this approach.
                 # Technically "Font Name" is a valid font. As it doesn't specify any font options
                 # though it is hard to match it correctly amongst the rest of the noise.
-                [[ -n "$binary" ]] &&
-                    term_font=$(
-                        strings "$binary" |
+                if [[ -n $(strings "$binary" | grep resource) ]]; then
+                    xrdb=$(xrdb -query)
+                    term_font=$(grep -im 1 -e "^${term/d}"'\**\.*font:' -e '^\*font:' <<< "$xrdb")
+                    term_font=${term_font/*"*font:"}
+                    term_font=${term_font/*".font:"}
+                    term_font=${term_font/*"*.font:"}
+                    term_font=$(trim "$term_font")
+                fi
+
+                if [[ -z $term_font && -n "$binary" ]]; then
+                    term_font=$(strings "$binary" | tac | \
```

Comment:
Why is the `tac` call needed? 

Also, it's better to use something more portable like awk: 

```sh
awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--] }'
```